linux_os/guide/system/bootloader-grub2/bios/group.yml

```diff
@@ -0,0 +1,8 @@
+documentation_complete: true
```

Comment:
In this case, let's not create new groups for grub2. The use of BIOS here is also the wrong technical connotation, and some of the grub checks are not UEFI specific and have been moved into the wrong groups anyway which shows and adds more confusion to the adding to new groups.

Reply:
Due to current limitations of the CPE support we have there is no other way than creating new groups, because the rules itself are already using `platform: machine`. So we have to create these groups so a new `platform` check can be applied. I have removed the rules that are not BIOS specific out from the group.

If BIOS is the wrong technical term, what should we put there?

Reply:
> Due to current limitations of the CPE support we have there is no other way than creating new groups, because the rules itself are already using `platform: machine`. So we have to create these groups so a new `platform` check can be applied. I have removed the rules that are not BIOS specific out from the group.

That's unfortunate. There are still some rules in this PR that are configured through `/etc/default/grub` and not in UEFI specific contexts.

> If BIOS is the wrong technical term, what should we put there?

While you do configure UEFI/Non-UEFI booting in the motherboard BIOS, people are going to read it wrong and think, "Let me go the the motherboard BIOS and configure..." I would use the term `non-UEFI` over BIOS to eliminate confusion some people are going to have.

Reply:
I have moved correctly the rules to the newly created `non-uefi` group. Also rebased on top of master. I think everything has been addressed.